app/components/markdown/markdown_code_block/markdown_code_block.js

```diff
@@ -61,7 +61,8 @@ class MarkdownCodeBlock extends React.PureComponent {
             animated: true,
             backButtonTitle: '',
             passProps: {
-                content: this.props.content
+                content: this.props.content,
+                theme: this.props.theme
```

Comment:
Why pass the theme in here? The Code screen can easily connect to the store itself and that would reduce coupling.

Reply:
it does not make sense to run `mapStateToProps` on that component every time the store changes just to get the theme

Reply:
But getTheme is memoized and we're calling it elsewhere anyway. This seems like it's adding additional coupling for virtually no benefit.